starcli/__main__.py

```diff
@@ -12,6 +15,8 @@
     status_actions,
 )
 
+CACHED_RESULT_FP = os.path.dirname(os.path.dirname(__file__)) + "/.cached_result.json"
```

Comment:
I don't think FP is a good constant name, how about:

```suggestion
CACHED_RESULT_PATH = os.path.dirname(os.path.dirname(__file__)) + "/.cached_result.json"
```